### PR TITLE
Update RPC `getversion`

### PR DESCRIPTION
**Pull request by mialbu**

### CODE REVIEW

File: core/src/main/java/io/neow3j/protocol/core/response/NeoGetVersion.java [hunk 1]
```diff
@@ -96,10 +103,54 @@ public String toString() {
...
         }
 
+        public static class Rpc {
+            @JsonProperty("maxiteratorresultitems")
+            private Integer maxIteratorResultItems;
+
+            @JsonProperty("sessionenabled")
+            private Boolean sessionEnabled;
+
+            public Rpc() {}
```
Comment (gsmachado) [inline core/src/main/java/io/neow3j/protocol/core/response/NeoGetVersion.java:118]: need an empty line after this.

Comment (thedanielmark) [reply core/src/main/java/io/neow3j/protocol/core/response/NeoGetVersion.java]: Done. Pls check.

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "java.compile.nullAnalysis.mode": "automatic"
```
Comment (gsmachado) [inline .vscode/settings.json:2]: why is this added to the code base? 👎

Comment (thedanielmark) [reply .vscode/settings.json]: Not sure when this was added but I've removed it.

Comment (gsmachado) [reply .vscode/settings.json]: all fine!

**Review by csmuller — APPROVED** [review]
Added a [commit](https://github.com/neow3j/neow3j/pull/1044/commits/5e87d222d20104881db06fab807a72a3ea8c8f31) for completeness.
 All good 👍
